README.md

```diff
@@ -79,8 +79,11 @@ For action helpers, this will mean better currying semantics:
   + [`underscore`](#underscore)
   + [`w`](#w)
 * [Array](#array-helpers)
+  + [`array`](#array)
```

Comment:
I can move it back, but I think it should be at the top because it's one of the most primitive helpers in this list. People need to know that it's available. I'm sure they'll come across it eventually but why not put it at the top so it's right there.